### PR TITLE
fix #1177 removed the delete button in layer properties in settings modal

### LINKED ISSUE #1177: Remove the delete button from the layer properties
With the latest introduction of the X button on the toc this is duplicated functionality and duplication is always bad.

I would remove it from tab.

x

### PR DESCRIPTION
**Pull request by saidaipparla**
delete

### CODE REVIEW

File: web/client/components/TOC/fragments/SettingsModal.jsx [hunk 1]
```diff
@@ -116,16 +115,6 @@ const SettingsModal = React.createClass({
             <Tab eventKey={3} title={<Message msgId="layerProperties.style" />} disabled>Tab 3 content</Tab>
           </Tabs>);
         const footer = (<span role="footer">
-            {this.props.includeCloseButton ? <Button bsSize={this.props.buttonSize} onClick={this.onClose}>{this.props.closeText}</Button> : <span/>}
```
Comment (mbarto) [inline web/client/components/TOC/fragments/SettingsModal.jsx]: Why did you also remove the close button?

File: web/client/components/TOC/fragments/SettingsModal.jsx [hunk 2]
```diff
@@ -116,16 +115,6 @@ const SettingsModal = React.createClass({
...
-            {this.props.includeDeleteButton ?
```
Comment (mbarto) [inline web/client/components/TOC/fragments/SettingsModal.jsx]: I would not remove this piece of code, just set includeDeleteButton to false in the plugin code.

Comment (saidaipparla) [inline web/client/components/TOC/fragments/SettingsModal.jsx]: ok i can do that

File: web/client/components/TOC/DefaultLayer.jsx [hunk 3]
```diff
@@ -42,6 +42,7 @@ var DefaultLayer = React.createClass({
         modalOptions: React.PropTypes.object,
         settingsOptions: React.PropTypes.object,
         visibilityCheckType: React.PropTypes.string,
+        includeDeleteButton: React.PropTypes.bool,
```
Comment (mbarto) [inline web/client/components/TOC/DefaultLayer.jsx]: includeDeleteButtonInSettings

File: web/client/components/TOC/DefaultLayer.jsx [hunk 4]
```diff
@@ -54,6 +55,7 @@ var DefaultLayer = React.createClass({
             activateRemoveLayer: false,
             activateLegendTool: false,
             activateSettingsTool: false,
+            includeDeleteButton: false,
```
Comment (mbarto) [inline web/client/components/TOC/DefaultLayer.jsx]: includeDeleteButtonInSettings

File: web/client/components/TOC/DefaultLayer.jsx [hunk 5]
```diff
@@ -94,6 +96,7 @@ var DefaultLayer = React.createClass({
                                {...this.props.settingsOptions}
                                hideSettings={this.props.hideSettings}
                                settings={this.props.settings}
+                               includeDeleteButton={this.props.includeDeleteButton}
```
Comment (mbarto) [inline web/client/components/TOC/DefaultLayer.jsx]: this.props.includeDeleteButtonInSettings

**Pull request closed without merging**